modules/ROOT/pages/pricing.adoc

```diff
@@ -2,86 +2,50 @@
+This limit doesn't impact applications configured with xref:cloudhub-2::ch2-configure-horizontal-autoscaling.adoc[Horizontal Pod Autoscaling in CloudHub 2.0]. You can continue to deploy applications if Horizontal Pod Autoscaling is enabled even if you hit the Global Resource Limits.
 
 == CloudHub 2.0 Application Deployment
 
-For organizations provisioned with a usage-based pricing package, CloudHub 2.0 uses replica sizing for application deployment. You can choose from the following replica sizes based on the application profile or size and display the equivalent in terms of legacy vCores. Replica sizes with the suffix `.mem` are memory-intensive options with the same compute. 
+For organizations provisioned with a usage-based pricing package, CloudHub 2.0 uses replica sizing for application deployment. You can choose replica sizes based on the application profile or size and display the equivalent in terms of legacy vCores. Replica sizes with the suffix `.mem` are memory-intensive options with the same compute. 
 
 include::partial$pricing.adoc[tag=pricingTable]
 
-During the monthly patching cycle, your applications automatically migrate to an equivalent replica size. Application deployment through the Mule Maven Plugin is not yet supported. However, you can deploy using the Runtime Manager UI and the Application Manager Experience APIs by leveraging a new field under `target.deploymentSettings` called `instanceType`.
+During the monthly patching cycle, your applications automatically migrates to an equivalent replica size. 
+
+Application deployment using the Mule Maven Plugin is not yet supported. To deploy your apps, use the Runtime Manager UI and the Application Manager Experience APIs with the new field in `target.deploymentSettings` called `instanceType`.
 
 == Access Management Entitlements
 
-Accounts provisioned with a usage-based pricing package show flows, messages, and throughput entitlements instead of vCores on Access Management. These entitlements are available at the root organization level and cannot be allocated among business groups. Because vCores are no longer the purchased entity, they can no longer be tracked or allocated among business groups. To view your monthly usage of flows, messages, and throughput, xref:general::usage-reports.adoc#access-usage-reports[access usage reports] by using the Access Management UI. 
+Accounts provisioned with a usage-based pricing package show flows, messages, and throughput entitlements in Access Management instead of vCores. These entitlements are available only at the root organization level. Because vCores are no longer the purchased entity, they can no longer be tracked or allocated among business groups. To view your monthly usage of flows, messages, and throughput, xref:general::usage-reports.adoc#access-usage-reports[access usage reports] in the Access Management UI. 
```

Comment:
This reads like vCores and IAM are equivalent. 

Accounts provisioned with a usage-based pricing package show flows, messages, and throughput entitlements,  instead of vCores, in Access Management.